kratos/solving_strategies/builder_and_solvers/explicit_builder.h

```diff
@@ -902,6 +902,12 @@ class ExplicitBuilder
 ///@name Type Definitions
 ///@{
 
+typedef TUblasSparseSpace<double> SparseSpaceType;
```

Comment:
Any guess?

Reply:
@roigcarlo

Reply:
to be honest I am a little bit lost, I am orbiting around the idea that pybinds tries to call the static mrComponents of the Kratoscomponents<ExplicitBuild> before that has been instantiated, the pybind bidding cannot find it, and everything crashes. But still have to do more iterations over this.

Expect it to keep failing :/

Reply:
Ok nevermind, you are registering in register_factory.h